src/MonitoringSolutions/docs/Remove-AzMonitorLogAnalyticsSolution.md

```diff
@@ -29,23 +29,22 @@ Deletes the solution in the subscription.
 
 ## EXAMPLES
 
-### Example 1: {{ Add title here }}
+### Example 1: Remove a monitor log analytics solution by name
 ```powershell
-PS C:\> {{ Add code here }}
+PS C:\> Remove-AzMonitorLogAnalyticsSolution  -ResourceGroupName azureps-manual-test -Name 'Containers(monitoringworkspace-2vob7n)'
 
-{{ Add output here }}
 ```
 
-{{ Add description here }}
+This command removes a monitor log analytics solution by name.
 
-### Example 2: {{ Add title here }}
+### Example 2: Remove a monitor log analytics solution by object
 ```powershell
-PS C:\> {{ Add code here }}
+PS C:\> $monitor = Get-AzMonitorLogAnalyticsSolution -ResourceGroupName azureps-manual-test -Name 'Containers(monitoringworkspace-pevful)'
+PS C:\> Remove-AzMonitorLogAnalyticsSolution -InputObject $monitor
```

Comment:
```suggestion
PS C:\> $monitor | Remove-AzMonitorLogAnalyticsSolution
```
We can sometimes write examples using pipeline syntax [more info](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_pipelines?view=powershell-7)

It's recommended and easier to use

Reply:
Add a new example description for this.